specification/translation/Azure.AI.DocumentTranslation/models.tsp

```diff
@@ -215,87 +190,6 @@ model Glossary {
   storageSource?: StorageSource;
 }
 
-@doc("""
-  Contains unified error information used for HTTP responses across any Cognitive
-  Service. Instances
-  can be created either through
-  Microsoft.CloudAI.Containers.HttpStatusExceptionV2 or by returning it directly
-  from
-  a controller.
-  """)
-@error
-model TranslationErrorResponse {
```

Comment:
So seems like this model wasnt used anywhere? Not in a template or anything?

Reply:
I was quite surprised too. Yes, this model is not used anywhere.